### ISSUE TITLE
Name 'iface' is not defined

### ISSUE THREAD

**Issue by CallMeLaNN** [adrianaksan/qgis-mongodb-plugin#8]
I'm new to QGIS. During installation of this plugin, I got name 'iface' is not defined.

**Comment by snodggu** [comment]
I too received this error. 
Thank you
![mongodb_plugin_iface_error](https://cloud.githubusercontent.com/assets/17085450/12850840/d85543f0-cbf4-11e5-84a7-5e28adffefc5.png)

**Comment by kyleestes** [comment]
I get this too. I'm running on Windows 7 x64 with QGIS 2.16.0 which reports Python 2.7.5. Any advice?

**Comment by adrianaksan** [comment]
Please note that the plugin has the following python dependencies: pymongo, json, ast.

Solution:
pip install pymongo, json, ast

Please advise.

**Comment by CallMeLaNN** [comment]
Today I checked my env, I got all pymongo, json, ast installed and I can import without any issue. I try to re-install this plugin. I can't recall when the error occurred but I can open the empty `Load MongoDB Points` window. I can't reproduce. I just fill in Server and Database and click Connect button but nothing is loaded. Never mind, my QGIS also has significant update and I already did a script to load data using pymongo and populate into layers.
